blockly-core/core/ui/block.js

```diff
@@ -254,14 +254,14 @@ Blockly.Block.isFreelyDragging = function() {
 
 /**
  * Wrapper function called when a mouseUp occurs during a drag operation.
- * @type {Array.<!Array>}
+ * @type {bindData}
```

Comment:
[It's declared in utils.js](https://github.com/code-dot-org/code-dot-org/pull/3079/files#diff-4f07ed683817ce12fcb89c3e30f708efR83) just above `Blockly.bindEvent_` (and I've never understood why that method is annotated private).

Somewhere I thought we had a convention of only using PascalCase for `new`able types, but I can't find it now.  I'll go ahead and make this change.